pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computeforwardingrule/globalcomputeforwardingrule/_http.log

```diff
@@ -848,11 +901,6 @@ X-Xss-Protection: 0
   "id": "000000000000000000000",
   "kind": "compute#forwardingRule",
   "labelFingerprint": "abcdef0123A=",
-  "labels": {
-    "cnrm-test": "true",
-    "label-one": "value-one",
-    "managed-by-cnrm": "true"
-  },
```

Comment:
This is the only field I want to discuss: seems like realGCP doesn't set labels during creation and then call `setLabels` request to set the labels. 

I think I need to double check how we handles the labels in direct controller, but I do not think that's a blocker, as the test passes and the labels has been updated.

Reply:
Addressed in #2711